assets/src/dashboard/app/views/shared/pageHeading.js

```diff
@@ -30,14 +30,9 @@ import {
   NavMenuButton,
   StandardViewContentGutter,
 } from '../../../components';
-import { SQUISH_CSS_VAR } from '../../../components/layout';
 import TypeaheadSearch from './typeaheadSearch';
 import TelemetryBanner from './telemetryBanner';
 
-const Container = styled.div`
-  padding: ${cssLerp('0px', '10px', SQUISH_CSS_VAR)} 0 0;
-`;
-
 const StyledHeader = styled.h2`
```

Comment:
Moved this padding to the end padding top value on `<Squish />` so that the start and end spacing can be in the same place.